### DISCUSSION TITLE
Welcome to Hyposyn Discussions!

### DISCUSSION THREAD

**Discussion by joecooldoo** [Hyposyn/Hyposyn#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by prisems** [comment]
hi

**Reply by joecooldoo** [reply]
why hello there :>

**Reply by prisems** [reply]
greetings

**Reply by joecooldoo** [reply]
hola

**Reply by joecooldoo** [reply]
lol translate

**Reply by prisems** [reply]
hola, como esta?

**Reply by joecooldoo** [reply]
Buenas gracias

**Reply by prisems** [reply]
tu vas a preguntar como estoy? brusca

**Reply by joecooldoo** [reply]
lo siento, ¿cómo estás?

**Reply by prisems** [reply]
soy muy bien

**Reply by prisems** [reply]
also im not using a translator so it may be rusty

**Reply by joecooldoo** [reply]
¡Eso es genial!

**Reply by joecooldoo** [reply]
oh nice i know a little bit of Spanish
I live in a place where a lot of people speak spanish

**Reply by prisems** [reply]
si, muy mucho

**Reply by prisems** [reply]
> oh nice i know a little bit of Spanish
> I live in a place where a lot of people speak spanish

i do to, i take classes for spanish

**Reply by joecooldoo** [reply]
and i dont lol

**Reply by joecooldoo** [reply]
hey you didnt change your github pfp

**Reply by prisems** [reply]
idk if I will

**Reply by joecooldoo** [reply]
why

**Reply by prisems** [reply]
idk man

**Reply by prisems** [reply]
![IMG_1016](https://user-images.githubusercontent.com/85964403/200909925-97135f96-8caa-41a4-88fd-69c2ddf4e14b.jpeg)

**Reply by joecooldoo** [reply]
i just use classic logo for any sites I don't use very often

**Reply by joecooldoo** [reply]
- [x] check list yay
- [x] wow

**Comment by prisems** [comment]
@joecooldoo 
im probably gonna make a 3D model or take one from one of my games (maybe I’ll remake the photo I sent) and do that cause I want to keep my model (as in words, on any game I have made, 3D, and you can look, I’ll link the images below, I say “The game must look good enough that the thumbnail may be a screenshot from the game”)
![IMG_0659](https://user-images.githubusercontent.com/85964403/200910631-26b2fc2c-2e27-4ede-af77-7aabfc855119.png)
![IMG_0651](https://user-images.githubusercontent.com/85964403/200910709-8c50b805-0059-4cc9-aac8-0d790d11260c.png)
![IMG_0652](https://user-images.githubusercontent.com/85964403/200910727-a6d670a3-0e80-49af-bbac-899a4bf98bb6.png)
![IMG_1020](https://user-images.githubusercontent.com/85964403/200911123-26d77783-61d5-46a8-b0c1-91b004d56cc4.png)
![IMG_1019](https://user-images.githubusercontent.com/85964403/200911134-d966d3fe-cdb3-4bbc-97cb-4bf38710833c.png)
![IMG_1018](https://user-images.githubusercontent.com/85964403/200911145-5184d392-6267-4318-a766-e91b3fdbc6cf.png)

some images may have sized wrong or other and may look as lower quality than actual*

**Reply by joecooldoo** [reply]
why are you showing me this exactly

**Reply by prisems** [reply]
forgot to say. Might make something like those or a model for my PFP lol

**Reply by joecooldoo** [reply]
um okay
